Makefile

```diff
@@ -32,14 +31,13 @@ inplace:
 	$(PYTHON) setup.py build_ext -i
 
 test-code:
-	$(NOSETESTS) -s nilearn $(NOSETESTS_OPTIONS)
+	python -m $(TEST_RUNNER) --pyargs nilearn -s -v --durations=0
 test-doc:
-	$(NOSETESTS) -s --with-doctest --doctest-tests --doctest-extension=rst \
-	--doctest-extension=inc --doctest-fixtures=_fixture `find doc/ -name '*.rst'`
+	python -m $(TEST_RUNNER) --pyargs nilearn -s -v --durations=0
```

Comment:
why has this rule become a duplicate of test-code? test-doc should do something different

Reply:
It needs to be rewritten to work with pytest. Will do that once test-code works